packages/web/tsup.config.js

```diff
@@ -17,6 +17,20 @@ export default defineConfig([
     },
     outDir: 'dist',
   },
+  {
+    ...cfg,
+    entry: {
+      index: 'src/nextjs/index.tsx',
+    },
+    external: ['react', 'next'],
+    outDir: 'dist/next',
+    esbuildOptions: (options) => {
+      // Append "use client" to the top of the react entry point
+      options.banner = {
+        js: '"use client";',
+      };
+    },
+  },
```

Comment:
**praise:** nice reuse of what you did for Speed insights.

I'm almost disappointed to see so few supported framework in Web Analytics 😉